docs/README.md

```diff
@@ -1,58 +1,44 @@
-# VTEX React App Template
+# VTEX Product List Context app
 
-Our guide repository to structure for react apps, that should be used as a template.
+This app is the Product List Context that should be used to store certain products. Example of usage: This is used on the `Shelf` app to store the visible products.
 
-We use `yarn` as our default package manager, before coding make sure to run yarn on: `root` and `react` folders.
+## API
 
-## Some features:
+`ProductListProvider`: Context provider that should be used to wrap your components that will use the hooks below.
+`useProductListState`: Hook that returns the state of the `ProductListStateContext`.
+Type of the State:
 
-### Tests
-
-For testing we use `@vtex/test-tools`, our own testing framework based on `react-testing-library`, the tests should be located on the `react/__tests__` folder. For references, visit our [repository](https://github.com/vtex/test-tools).
-
-### Hooks
-
-Husky hooks tha runs on every `pre-commit` and `pre-push`.
-
-### Intl Equalizer
-
-Tool for equalizing the messages located on the `messages` folder/builder. It's configured to use the **en.json** as the default file for comparison. For references, visit our [repository](https://github.com/vtex/intl-equalizer).
-
-### Lint + Formatting
-
-TS lint configured with Prettier and .Config.
-
-### Available Scripts
-
-```json
-{
-  "lint": "cd ./react && yarn lint",
-  "test": "cd ./react && yarn test",
-  "lint:locales": "intl-equalizer",
-  "locales:fix": "intl-equalizer --fix",
-  "verify": "yarn lint && yarn lint:locales && yarn test"
+```
+interface State {
+  products: any[]
```

Comment:
The name doesn't make it clear the purpose of this variable.

Maybe change it to `visibleProducts`